src/service/fetchURL/individualEntity/fetchIndividualEntity.ts

```diff
@@ -1,6 +1,6 @@
 export const fetchIndividualEntity:any = async(entityID:string) => {
     try {
-        const response = await fetch(`${process.env.REACT_APP_DATABASE_URL}carbonmap/ccm-front-end/master/dummy_data/${entityID}`);
+        const response = await fetch(`${process.env.REACT_APP_DATABASE_URL}${entityID}`);
```

Comment:
Should this have a "/" between `REACT_APP_DATABASE_URL` and `entityID` ?

Reply:
No, entityID should be passed in with "/" at the begging of the string